src/components/EditableField/Group/Group.md

```diff
@@ -140,6 +140,110 @@ const [secondDefaultValue, setSecondDefaultValue] = React.useState(1)
 );
 ```
 
+### Example With Autocomplete
+
+##### <span style="color: red">Attention:</span> do not use <span style="color: red">saveOnBlur</span> or <span style="color: red">resetOnBlur</span> when using the Autocomplete with EditableField.Groups, the Autocomplete behavior maintain the buttons until the user interacts with them.
```

Comment:
I think something like this may be clearer to what happens:
"Attention: when an `EditableField.Group` has an `EditableField.Autocomplete` inside of it, the `saveOnBlur` and `resetOnBlur` props are ignored, and the group behaves as is both props are `false` (so no save or reset will happen on blur)"